src/main/java/org/broadinstitute/hellbender/utils/genotyper/AlleleLikelihoods.java

```diff
@@ -446,7 +445,7 @@ private void normalizeLikelihoodsPerEvidence(final double maximumBestAltLikeliho
      * @return never {@code null}, but with {@link BestAllele#allele allele} == {@code null}
      * if non-could be found.
      */
-    private BestAllele searchBestAllele(final int sampleIndex, final int evidenceIndex, final boolean canBeReference, Optional<double[]> priorities) {
+    private BestAllele searchBestAllele(final int sampleIndex, final int evidenceIndex, final boolean canBeReference, final double[] priorities) {
```

Comment:
My IntelliJ complains about using Optional on a method argument... don't know why that is smelly. In any case a nullable value covers the same functionality nicely and is more efficient, I guess.